.env.example

```diff
@@ -0,0 +1,22 @@
+LADT_MODEL_DIR="models" # Name of persistant storage directory that models will be saved to
+LADT_ELASTICSEARCH_ENDPOINT='' # address to Elasticsearch endpoint
+LADT_MODEL='map.sav' # path to SOM map to update
+LADT_INDEX='logstash-' # name of Elasticsearch index to pull logs from
+LADT_TIME_SPAN=900 # number of seconds from now that you would like to query Elasticsearch for training logs
+LADT_MAX_ENTRIES=450000 # limits the number of entries returned from Elasticsearch for the training query
+LADT_ITERS=45000 # number of training iterations of the SOM
+LADT_SERVICE='journal' # name of Elasticsearch service to be monitored
+LADT_TRAIN_LAG=15 # Number of Inference iterations between retraining the model
+LADT_UPDATE_MODEL=True
+
+LADI_ELASTICSEARCH_ENDPOINT='' # address to Elasticsearch endpoint for inference
+LADI_MODEL='map.sav' # path to SOM map to test against
+LADI_INDEX='logstash-' # name of Elasticsearch index to pull logs from
+LADI_TARGET_INDEX='mcliffor_test_ingest-' # name of index the anoamliy data will be pushed back to in Elasticsearch
+LADI_TIME_SPAN=60 # number of seconds from now that you would like to query Elasticsearch for
+LADI_MAX_ENTRIES=35000 # limits the number of entries returned from Elasticsearch for the inference query
+LADI_SERVICE='journal' # name of Elasticsearch service to be monitored
+LADI_THRESHOLD=1.5 # float tunes the anomaly threshold based on the max value of the training set
+
+LOCAL_DATA_FILE=validation_data/Hadoop_2k.json
+LOCAL_RESULTS_FILE=/tmp/local-results.txt
```

Comment:
@MichaelClifford Are we still using this file? I think you set some of these env vars with safe defaults

Reply:
@zmhassan, env.sh was a script used early on to manage and quickly load all of the needed environment variables. But,  AFAIU @vpavlin moved all of these into `config.py` or the appropriate storage class. So, I do not think this file is being used any more.

Reply:
It is definitely not used by OpenShift, but what @durandom is proposing IIUC is to use it in Travis

Reply:
@durandom what do you think. Should we delete this file?

Reply:
what file are you all talking about? 
`LOCAL_RESULTS_FILE` ? 
or `env.sh`

The former seems to be still used.
And `config.py` is still using env variables.